src/style.css

```diff
@@ -1,3 +1,64 @@
+:root {
+  --blue: #00acdc;
+}
+
 body {
+  font-family: Roboto, Helvetica, Arial, sans-serif;
+  margin: 0;
+  background-color: aqua;
```

Comment:
Avoid using color names like `aqua`. It's better to use RGB or HEX color codes for consistency and better control over the color shades.